bitnami/etcd/Chart.yaml

```
@@ -24,4 +24,4 @@ name: etcd
 sources:
   - https://github.com/bitnami/bitnami-docker-etcd
   - https://coreos.com/etcd/
-version: 8.1.1
+version: 8.1.2
```

Since this is adding a new feature, we should increase the minor instead:

```suggestion
version: 8.2.0
```